icons/mithril/mithril-line.svg

```diff
@@ -1 +1 @@
056-42.485-42.492 0-.013.012-.087.012-.215v-.281l-.24-.147C8.735 74.362.5 60.908.5 45.532.5 22.094 19.556 3.038 42.988 3.038Zm0 6.585c-19.794 0-35.906 16.108-35.906 35.906 0 11.66 5.614 22.011 14.252 28.574l.63.479.163-.775c2.474-11.908 9.973-21.97 20.15-27.96l.244-.144.002-.285c.036-12.797 5.767-24.254 14.768-32.03l.604-.52-.73-.317a35.606 35.606 0 0 0-14.177-2.928Zm42.02 0a35.578 35.578 0 0 0-14.172 2.928l-.73.316.603.522c9.028 7.795 14.775 19.294 14.775 32.136 0 .004-.015.069-.015.215v.281l.24.147c10 6.085 17.325 16.139 19.68 27.986l.152.77.633-.463c8.915-6.54 14.74-17.06 14.74-28.932-.004-19.794-16.112-35.906-35.906-35.906Zm-42.02 1c4.492-.004 8.918.93 13.073 2.623-8.784 7.887-14.41 19.224-14.522 31.908-10.045 6-17.484 15.887-20.17 27.61C13.322 66.365 8.082 56.588 8.082 45.53c0-19.256 15.654-34.906 34.906-34.906Zm42.02 0c19.252 0 34.902 15.654 34.906 34.906 0 11.283-5.456 21.245-13.795 27.631-2.559-11.661-9.777-21.556-19.637-27.654-.005-12.83-5.675-24.302-14.543-32.26 4.154-1.693 8.579-2.628 13.069-2.623ZM63.996 16.48l-.297.22c-7.89 5.84-13.328 14.808-14.394 25.081l-.08.78.74-.256c4.321-1.488 8.943-2.332 13.76-2.332 5.031 0 9.84.922 14.322 2.54l.744.268-.078-.787c-1.014-10.36-6.477-19.411-14.418-25.293Zm0 1.295c7.353 5.577 12.411 13.966 13.568 23.575-4.354-1.482-8.983-2.377-13.84-2.377-4.639 0-9.079.82-13.269 2.175 1.201-9.526 6.237-17.834 13.541-23.373zm-.271 28.788a35.67 35.67 0 0 0-14.022 2.857l-.346.146.045.373c1.233 9.991 6.586 18.708 14.297 24.424l.297.221.299-.22c7.652-5.673 12.974-14.286 14.262-24.178l.047-.37-.34-.152a35.641 35.641 0 0 0-14.54-3.102Zm-.002 1h.002c4.734 0 9.403 1.01 13.746 2.89-1.324 9.305-6.31 17.399-13.473 22.836-7.22-5.48-12.238-13.671-13.512-23.07 4.201-1.733 8.69-2.66 13.237-2.657Zm-20.534 5.5-.632.464c-7.72 5.671-13.095 14.329-14.395 24.284l-.049.369.34.152a35.737 35.737 0 0 0 14.535 3.098c5.03 0 9.826-1.047 14.176-2.928l.732-.316-.603-.522c-7.032-6.075-12.07-14.382-13.95-23.832Zm41.563.347-.162.778c-1.937 9.306-6.941 17.48-13.885 23.48l-.603.521.732.317a35.656 35.656 0 0 0 14.172 2.928c4.97 0 9.709-1.02 14.023-2.858l.346-.146-.045-.373c-1.212-9.845-6.426-18.44-13.947-24.166Zm-42.156 1.42c1.985 9.046 6.787 17.005 13.445 22.984-4.04 1.644-8.427 2.615-13.05 2.616-4.734-.003-9.403-1.011-13.745-2.887 1.32-9.247 6.267-17.276 13.35-22.713zm42.732.37c6.89 5.48 11.68 13.444 12.916 22.58-4.092 1.687-8.549 2.654-13.238 2.654h-.002c-4.482.002-8.898-.93-13.045-2.616 6.567-5.9 11.332-13.722 13.37-22.619ZM63.996 82.421l-.246.139a42.13 42.13 0 0 1-20.762 5.459c-5.032 0-9.837-.923-14.322-2.538l-.744-.267.078.787c1.787 18.142 17.126 32.37 35.725 32.37 18.53 0 33.828-14.12 35.71-32.16l.083-.778-.74.254c-4.328 1.486-8.944 2.332-13.768 2.332a42.173 42.173 0 0 1-20.768-5.46zm0 1.129a43.161 43.161 0 0 0 21.014 5.47c4.65 0 9.088-.821 13.285-2.178-2.17 17.175-16.806 30.53-34.57 30.53-17.83 0-32.503-13.456-34.586-30.73 4.36 1.481 8.989 2.378 13.85 2.378a43.117 43.117 0 0 0 21.007-5.47z" color="#000" style="-inkscape-stroke:none"/></svg>
```

Comment:
Nice! The icon looks good, but the code is unoptimized. Please optimize it :)
You can do so with SVGOMG: https://github.com/devicons/devicon/issues/1279#issuecomment-1195433256